portfolio/src/main/webapp/script.js

```diff
@@ -11,3 +11,19 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+$(document).ready(function(){
+    $('#2048-pic').hide();
```

Comment:
You generally wouldn't want to hide this in JS - if there's a problem with the JS (it didn't load, user's browser disables or has an issue with your code, etc), then everything is shown. It also means there's a delay - it isn't hidden until the script runs.

Instead, the general recommendation is to set it to display:none initially in CSS.